scan/lib/scan/runner.rb

```diff
@@ -122,9 +122,10 @@ def test_results
       # We'll have to regenerate from the xcodebuild log, like we did before version 2.34.0.
       UI.message("Generating test results. This may take a while for large projects.")
 
-      reporter_options_generator = XCPrettyReporterOptionsGenerator.new(false, [], [], "", false)
+      reporter_options_generator = XCPrettyReporterOptionsGenerator.new(false, [], [], "", false, "")
```

Comment:
Should this be a `nil` instead of an empty string? The tests use a `nil` when not passing anything but this uses any empty string 🤔 Maybe we also add a tests for `""` so we know that works as expect too? I'm okay with either of these solutions 😊

Reply:
Good point. I've made this change and added one spec for nil, and updated 4 of the existing 10 `XCPrettyReporterOptionsGenerator` specs  to use `""` instead of `nil` to validate that it doesn't impact the other reporter options - I hope that's OK and not considered corner-cutting! Happy to go the long way round if you think it's better. 😊